.github/workflows/ci-sync-tests-gcp.yml

```diff
@@ -0,0 +1,616 @@
+name: CI Docker
```

Comment:
This is waiting for a successful full sync, which probably needs some other fixes.

Reply:
I really think we should do a full sync, unless this change is urgent?
If we make multiple significant changes to CI, it will be hard to work out which code is causing any failures.

A Zebra testnet and lightwalletd mainnet full sync should be enough, both of them complete within about 8 hours.